application/resources/views/admin/product_categories/index.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('layouts.admin')
+
+@section('content')
+    <div class="col-12 mt-4">
+        <form class="form-signin" method="GET" action="{{ route('admin.product_categories.index') }}">
+            @csrf
+            <div class="border row center-block text-center">
+                <div class="col-12 mt-4">
+                    <input type="name" id="name" class="form-control @error('name') is-invalid @enderror" name="name" value="{{ $old['name'] }}" placeholder="名称" autofocus>
```

Comment:
こんな変数渡さなくても
```suggestion
                    <input type="name" id="name" class="form-control @error('name') is-invalid @enderror" name="name" value="{{ request('name') }}" placeholder="名称" autofocus>
```
でとれます